s4e-backend/src/main/java/pl/cyfronet/s4e/search/SearchApiParams.java

```diff
@@ -0,0 +1,58 @@
+package pl.cyfronet.s4e.search;
```

Comment:
I'd move that to a seprate package. Maybe `s4e.dhus.api` or something like that. `s4e.search` is meant more for our internal search.